packages/cli/src/generate.ts

```diff
@@ -181,6 +187,7 @@ export async function generate(
           scalars: scalarTypes,
           namingConvention: 'keep',
           enumsAsTypes: enumsAsStrings,
+          enumsAsConst: enumsAsConst,
```

Comment:
use the same logic that `enumsAsStrings` has to use the value from config, and then fallback to the default config